src/main/java/Main/Main.java

```diff
@@ -8,41 +11,47 @@ public static void main(String[] args) throws InterruptedException {
 
         //This code is as shitty as it can be. Don't roast me yet please
         Scanner scanner = new Scanner(System.in);
-        System.out.println("U see big classroom with many chairs. On almost every chair sits one student. In this moment you recalled that today is exam day! In whole classroom only one chair is free. ");
-        System.out.println("Bad news. It's the chair in front of your professor. What will you do young student? ");
-        System.out.println("[W] - Sit in this chair and try to pass this exam (You didn't attend this lecture)");
-        System.out.println("[S] - Pretend that you have made a mistake and run as fast as you can");
+        ConsolePrinter consolePrinter = new ConsolePrinter();
```

Comment:
What if i would like to use another printer? Remember about inversion of control.